amundsen_common/fixtures.py

```diff
@@ -0,0 +1,224 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+import string
+from typing import Any, List, Optional
+
+from amundsen_common.models.table import (Application, Column,
+                                          ProgrammaticDescription, Stat, Table,
+                                          Tag)
+from amundsen_common.models.user import User
+
+
+class Fixtures:
+    counter = 1000
```

Comment:
It might be useful to add some documentation somewhere on how to use this Fixtures class (a summary of some kind so that people don't have to read all the code to figure it out)

Reply:
- [ ] document fixtures